packages/flutterfire_ui/lib/src/i10n/lang/pt.dart

```diff
@@ -0,0 +1,263 @@
+  @override
+  final String signInHintText;
+  @override
+  final String signOutButtonText;
+  @override
+  final String phoneInputLabel;
+  @override
+  final String phoneNumberIsRequiredErrorText;
+  @override
+  final String phoneNumberInvalidErrorText;
+  @override
+  final String profile;
+  @override
+  final String name;
+  @override
+  final String deleteAccount;
+  @override
+  final String passwordIsRequiredErrorText;
+  @override
+  final String confirmPasswordIsRequiredErrorText;
+  @override
+  final String confirmPasswordDoesNotMatchErrorText;
+  @override
+  final String confirmPasswordInputLabel;
+  @override
+  final String forgotPasswordButtonLabel;
+  @override
+  final String forgotPasswordViewTitle;
+  @override
+  final String resetPasswordButtonLabel;
+  @override
+  final String verifyItsYouText;
+  @override
+  final String differentMethodsSignInTitleText;
+  @override
+  final String findProviderForEmailTitleText;
+  @override
+  final String continueText;
+  @override
+  final String countryCode;
+  @override
+  final String codeRequiredErrorText;
+  @override
+  final String invalidCountryCode;
+  @override
+  final String chooseACountry;
+  @override
+  final String enableMoreSignInMethods;
+  @override
+  final String signInMethods;
+  @override
+  final String provideEmail;
+  @override
+  final String goBackButtonLabel;
+  @override
+  final String passwordResetEmailSentText;
+  @override
+  final String forgotPasswordHintText;
+  @override
+  final String emailLinkSignInButtonLabel;
+  @override
+  final String signInWithEmailLinkViewTitleText;
+  @override
+  final String signInWithEmailLinkSentText;
+  @override
+  final String sendLinkButtonLabel;
+  @override
+  final String arrayLabel;
+  @override
+  final String booleanLabel;
+  @override
+  final String mapLabel;
+  @override
+  final String nullLabel;
+  @override
+  final String numberLabel;
+  @override
+  final String stringLabel;
+  @override
+  final String typeLabel;
+  @override
+  final String valueLabel;
+  @override
+  final String cancelLabel;
+  @override
+  final String updateLabel;
+  @override
+  final String northInitialLabel;
+  @override
+  final String southInitialLabel;
+  @override
+  final String westInitialLabel;
+  @override
+  final String eastInitialLabel;
+  @override
+  final String timestampLabel;
+  @override
+  final String latitudeLabel;
+  @override
+  final String longitudeLabel;
+  @override
+  final String geopointLabel;
+  @override
+  final String referenceLabel;
+
+  const PtLocalizations({
+    this.emailInputLabel = 'E-mail',
+    this.passwordInputLabel = 'Senha',
+    this.signInActionText = 'Fazer login',
+    this.registerActionText = 'Registrar',
+    this.signInButtonText = 'Fazer login',
+    this.registerButtonText = 'Registrar',
+    this.linkEmailButtonText = 'Próximo',
+    this.signInWithPhoneButtonText = 'Fazer login com o telefone',
+    this.signInWithGoogleButtonText = 'Fazer login com o Google',
```

Comment:
```suggestion
    this.signInWithGoogleButtonText = 'Entrar com Google',
```

Reply:
What do you think, `Login com o Google?`
So the context `login` is maintained.

Otherwise, `Entrar com o Google`, it's ok for me.

Reply:
Well..I have the impression that is generally used just 'Entrar com $marca' (Google, Facebook, Apple). As per English definition, it's not like 'Sign in with the Google'. Could be some branding approach to use without the 'o'.
 Login context is good too.
 Actually, it's just a small detail and personal preference maybe, really not a big deal.
 Anyways, you've done a great job, it's up to you to decide.
 Would be great if next update this translation could be included.

Reply:
So, the translate this, I followed the Firebase UI on Android https://github.com/firebase/FirebaseUI-Android/blob/master/auth/src/main/res/values-pt-rBR/strings.xml

Here is, `Fazer login com o Google` -> `Sign in with Google` 

Thanks for the reviews.